cves/CVE-2019-14232.yml

```diff
@@ -140,8 +154,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    An  automated testing tool with an extensive list of inputs would 
+    have caught this potential Denial of Service.
```

Comment:
Maybe elaborate on what the automated attack would look like in terms of input and the system response